rpm/sailfish-browser.spec

```diff
@@ -22,6 +22,8 @@ BuildRequires:  qt5-qttools
 BuildRequires:  qt5-qttools-linguist
 BuildRequires:  gdb
 BuildRequires:  oneshot
+BuildRequires:  gtest-devel
+BuildRequires:  libgmock-devel
```

Comment:
The main consumer of this info (apart from the build robots) is not a developer, but a package maintainer. In her perverted logic she builds the source package `sailfish-browser` that results in some build artifacts including coincidentally the binary package `sailfish-browser`. :)  This binary package can't be built unless all build deps are met. And it may save significant amount of time when hunting down build fails in the build machinery (e.g. when a package that we depend on gets renamed) if all the build deps are not scattered around the spec which is difficult to scan quickly due its size. Debian, for example, forces build deps to be included in the Source section of debian/control and for a good reason.
Probably I've spent too much time on package maintenance :)